### ISSUE TITLE
[FR] Don't insert images with external link format

### ISSUE THREAD

**Issue by RenanSantos7** [aleksey-rezvov/obsidian-local-images#54]
Instead of create external links (`![Alt text](path/to/file/image.png)`) insert images just in the Obsidian format (`![[image.png]]`).

The first option gets in the way because the images appear in Graph View as if they were external files.

**Comment by ellavs** [comment]
Yes, this is very necessary, I use external links in MD format (not in Obsidian format) and the plugin does not work :(

**Comment by ksdavidc** [comment]
hear hear
